lms/tasks/email_digests.py

```diff
@@ -46,7 +46,9 @@ def send_instructor_email_digest_tasks():
     """
     now = datetime.now(timezone.utc)
     weekday = EmailPrefs.DAYS[now.weekday()]
-    created_before = datetime(year=now.year, month=now.month, day=now.day, hour=5)
+    created_before = datetime(
```

Comment:
Make this timezone aware, it should propagate down the line.

Reply:
So this should mean that timezone-aware datetime strings begin to be written to the `task_done` table :+1: